test/checks/visibility/hidden-content.js

```diff
@@ -0,0 +1,36 @@
+describe('hidden content', function () {
```

Comment:
Add tests to make sure you're not selecting descendants of elements with `display:none` / `visibility:hidden`.